### PR TITLE
PRESIDECMS-1687: new boolean badge renderer (admin only)

### PR DESCRIPTION
**Pull request by jjannek**

### CODE REVIEW

**Comment by DominicWatson** [comment]
This is awesome thanks - will look at merging next week :) thanks @jjannek

File: system/handlers/renderers/content/BooleanBadge.cfc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		
+		var booleanBadgeValue = presideObjectService.getObjectPropertyAttribute( objectName=objectName, propertyName=propertyName, attributeName=valueAttribute, defaultValue="" );
+		var booleanBadgeStyle = presideObjectService.getObjectPropertyAttribute( objectName=objectName, propertyName=propertyName, attributeName=styleAttribute, defaultValue="info" );
+		
+		if ( isEmpty( booleanBadgeValue ) ) {
+			return "";
+		}
+		
+		booleanBadgeValue = translateResource( uri="preside-objects.#objectName#:field.#propertyName#.#valueAttribute#", defaultValue=booleanBadgeValue );
+		booleanBadgeValue = translateResource( uri="preside-objects.#objectName#:field.#propertyName#.#styleAttribute#", defaultValue=booleanBadgeStyle );
```
Comment (sebduggan) [inline system/handlers/renderers/content/BooleanBadge.cfc]: Empty check need to move down below the i18n logic...

Comment (jjannek) [reply system/handlers/renderers/content/BooleanBadge.cfc]: fixed

**Comment by DominicWatson** [comment]
To be released in 10.11.6

**Pull request merged**